clients/drcachesim/tracer/raw2trace.h

```diff
@@ -876,6 +882,9 @@ template <typename T> class trace_converter_t {
             DR_CHECK(reinterpret_cast<trace_entry_t *>(buf) == buf_base,
                      "We shouldn't have buffered anything before calling "
                      "append_bb_entries");
+            if (saw_filter_endpoint && true) {
+                impl()->set_file_type(tls, OFFLINE_FILE_TYPE_DEFAULT);
```

Comment:
But what if some other feature is in effect like _NO_OPTIMIZATIONS or _INSTRUCTION_ONLY?  This needs to preserve those other flags.

Reply:
I fixed this to preserve the flags. I had to cast it to int to clear the filter flags,